GUI/TaskWindows/SearchTaskWindow.xaml

```diff
@@ -445,6 +445,7 @@
                                     Output of .mzID requires protein parsimony. Please check "Apply protein parsimony and construct protein groups".
                                 </ToolTipService.ToolTip>
                             </CheckBox>
+                            <CheckBox x:Name="IncludeMotifInModNamesCheckBox" Margin="20 0 0 0" Content="Include motif in mod names" IsEnabled="{Binding IsChecked, ElementName=CkbMzId}" />
```

Comment:
Can you add a tool tip explaining this is just for mzID?
Maybe put this checkbox below the decoy/contaminant checkboxes, which are likely more relevant to most users.